tests/test_annoy_console.py

```diff
@@ -0,0 +1,23 @@
+"""module contains tests for the Slog class, specifically the annoying method, which is responsible for slog annoying"""
+
+from slogpy.slog import Slog as slog  # noqa: N813,
+
+
+def test_annoy_unique_message(capsys, msg="This is a annoy test"):  # noqa: D103
+    """Given: a single line annoy message
+
+    When: passed into slog.annoy
+    Then: method will check for annoy message only once and display it.
+    """
+    # Arrange
+
+    # Act
+    slog.annoy(msg)
```

Comment:
Since you are only calling annoy() once, it is impossible to tell if annoy() if fulfilling its promise to only annoy once.